crates/tinyvec-embed/src/db.rs

```diff
@@ -244,6 +252,18 @@ impl Value {
     }
 }
 
+impl TryInto<usize> for &Value {
+    type Error = Error;
+
+    fn try_into(self) -> Result<usize> {
+        if let Value::Number(n) = self {
+            Ok(n.clone() as usize)
```

Comment:
I think f32 implements `Copy`
```suggestion
            Ok(n as usize)
```

Reply:
This is &f32 not f32, because we're implementing for &Value not Value, so I guess it's still needed no ?

Reply:
Ah sorry missed that you were implementing on `&Value` & not `Value`. Perhaps you should switch to:
```diff
-impl TryInto<usize> for &Value
+impl TryInto<usize> for Value
```

Reply:
meta.get returns a &Value, that's why I implemented it this way. It seems that inner_string also applies to a &Value, so this also add some consistency between all Value conversion methods, what do you think ?